app/src/main/java/com/google/samples/apps/sunflower/data/PlantRepository.kt

```diff
@@ -16,6 +16,7 @@
 
 package com.google.samples.apps.sunflower.data
 
+import kotlinx.coroutines.flow.Flow
```

Comment:
Is this import needed? I'd also add a comment about threading in this class.

e.g. `// Collecting from the flows in Dao is main-safe as Room's coroutines support moves the execution of the query off the main thread.`

Reply:
Missed that unused import.  Good call on the comment - added in.